src/views/RegisterPage/index.js

```diff
@@ -27,10 +27,41 @@ class RegisterPage extends Component {
   }
 
   handleChange = event => {
-    this.setState({[event.target.name]: event.target.value});
+    if(event.target.name == "terms"){
+        if(event.target.checked){
```

Comment:
this can be condensed into:
`this.setState({
    [event.target.name]: !event.target.checked });
}`